packages/pds/src/app-view/api/app/bsky/unspecced.ts

```diff
@@ -70,19 +77,10 @@ export default function (server: Server, ctx: AppContext) {
         requester,
       )
 
-      const noRecordEmbeds = feed.map((post) => {
-        delete post.post.record['embed']
-        if (post.reply) {
-          delete post.reply.parent.record['embed']
-          delete post.reply.root.record['embed']
-        }
-        return post
-      })
```

Comment:
Records that contain image or external embeds will still show-up here— is this logic just not needed at all anymore?  I recall it goes back to some issues with lexicon validation and the client.

Reply:
oh shit that's right... okay this does need to be back in my bad